Python_Scripts/compareStateTrajectories_BioModels_1.py

```diff
@@ -1,178 +0,0 @@
-"""Simulate trajectories for all biomodels models and load reference
```

Comment:
I really like how much red I see here :partying_face:

Reply:
yeah... it's all gone...